### ISSUE TITLE
Unknown garbage collector name

### ISSUE THREAD

**Issue by AllenShi** [elastic/logstash#8782]
- Version:
5.6.4

- Operating System: 
Ubuntu 16.04

- Java:
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 8.0.5.0 - pxa6480sr5-20170905_01(SR5))
IBM J9 VM (build 2.9, JRE 1.8.0 Linux amd64-64 Compressed References 20170901_363591 (JIT enabled, AOT enabled)
J9VM - d56eb84
JIT  - tr.open_20170901_140853_d56eb84
OMR  - b033a01)
JCL - 20170823_01 based on Oracle jdk8u144-b01

- Config File (if you have sensitive info, please remove it):
- Sample Data:
- Steps to Reproduce:

   Start Logstash on IBM JRE 8, the following error is shown in logstash-plain.log.

```
[2017-11-30T05:28:05,150][ERROR][logstash.instrument.periodicpoller.jvm] Unknown garbage collector name {:name=>nil}
[2017-11-30T05:28:05,150][ERROR][logstash.instrument.periodicpoller.jvm] Unknown garbage collector name {:name=>nil}
[2017-11-30T05:28:10,155][ERROR][logstash.instrument.periodicpoller.jvm] Unknown garbage collector name {:name=>nil}
```

**Comment by jordansissel** [comment]
Hello! Thank you for the report. We do not test on IBM jdk. Looks like a
bug in the ibm jdk that we cannot query the garbage collector setting? I
have not investigated.

**Comment by jakelandis** [comment]
Fixed with PR https://github.com/elastic/logstash/pull/8784

**Comment by AllenShi** [comment]
@jordansissel, Fixed with PR #8784. The garbage collector setting can be queried, but it's listed in your validation list.
